SwiftLeeds/Views/About/SponsorGridView.swift

```diff
@@ -9,13 +9,14 @@ import SwiftUI
 
 struct SponsorGridView: View {
     @Environment(\.openURL) var openURL
-    let goldSponsors = Sponsor.platinum
 
     var body: some View {
         VStack(spacing: Padding.cellGap) {
             sectionHeader(text: "Platinum Sponsors")
-            contentTile(for: Sponsor.codemagic)
-            contentTile(for: Sponsor.stream)
+            ForEach(Sponsor.platinum, id: \.self) { sponsor in
```

Comment:
Won't this show the sponsors twice as it's a mix of the for each and constant tiles?

Reply:
sorted, missed committing a chunk